core/src/test/java/org/bitcoinj/net/discovery/DnsDiscoveryTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Tim Strasser
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.net.discovery;
+
+import org.bitcoinj.params.MainNetParams;
+import org.junit.Test;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class DnsDiscoveryTest {
+
+    @Test
```

Comment:
These tests probably need the network. I'm happy to merge them, but they need to be excluded from regular build. See build.gradle for that.